src/main/riddl/Members/members.riddl

```diff
@@ -313,21 +292,13 @@ context MembersContext  is {
                     and "set entity lastModifiedOn to today()"
                     and "yield event MemberInfoUpdated"
                 }
-                on command ActivateMember {
-                  then error "Cannot activate a member that is already activated"
-                }
-                on command InactivateMember {
-                  then "morph Member entity to Inactive state"
-                }
-                on command SuspendMember {
-                  given "entity has suspensionReason non-empty"
-                  then "morph Member entity to Suspended state"
-                  and "set entity lastModifiedBy to actingMember from message"
-                  and "set entity lastModifiedOn to today()"
-                  and "yield event MemberSuspended"
+                on command UpdateMemberStatus {
```

Comment:
All three actions of the example's "then" part (below) should be specific operators, not arbitrary (string) ones. Please use the few operators we have when appropriate.  Furthermore, these three are not appropriate. You've opted for enumerator based state changes which means:
* You need to set the state value in a "MemterState" (the enumeration type) valued field
* Change the handler with the "become" operator.  Using "morph" means you're changing to a different RIDDL State